src/BuildingRegistry.Api.Legacy.Abstractions/Converters/BuildingUnitExtensions.cs

```diff
@@ -141,17 +142,25 @@ public static class BuildingUnitFunctionExtensions
 
         public static GebouweenheidFunctie? Map(this BuildingUnitFunction function)
         {
-            if (BuildingUnitFunction.Common == function)
-            {
-                return GebouweenheidFunctie.GemeenschappelijkDeel;
-            }
-
-            if (BuildingUnitFunction.Unknown == function)
-            {
-                return GebouweenheidFunctie.NietGekend;
-            }
-
-            throw new ArgumentOutOfRangeException(nameof(function), function, null);
+            var dictionary = new Dictionary<BuildingUnitFunction, GebouweenheidFunctie>
```

Comment:
Don't know? Then we allways instantiate the dictionary even when possibly not needed...

Reply:
Its static data, its required for business so why not instantiate it once at the start of the application rather than each time you are mapping properties? You will use this dictionary during the lifetime of the service, this is just about reducing the instantiation costs when we can.